config/environments/production.rb

```diff
@@ -24,7 +24,11 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
Wow, do we compile assets on production for every request? This adds a lot of overhead...